test/e2e_node/privileged_test.go

```diff
@@ -151,7 +150,6 @@ func (config *PrivilegedPodTestConfig) dialFromContainer(containerIP string, con
 	var output map[string]string
 	err = json.Unmarshal([]byte(stdout), &output)
 	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Could not unmarshal curl response: %s", stdout))
-	glog.Infof("Deserialized output is %v", output)
```

Comment:
I copied this from the original test. The output seems pretty meaningful. E.g., `"privileged_test.go:154] Deserialized output is map[output:RTNETLINK answers: Operation not permitted"` 

Do we have any general rules for logging in a node e2e suite? A cluster e2e test logs quite a lot at the info level, but I've just noticed that it's very different in the node e2e suite.